Mathlib/Combinatorics/SimpleGraph/Walk.lean

```diff
@@ -1242,4 +1348,58 @@ theorem map_toDeleteEdges_eq (s : Set (Sym2 V)) {p : G.Walk v w} (hp) :
 
 end Walk
 
+namespace Walk
+
+open Classical
+variable {G}
+
```

Comment:
```suggestion
```
Can you remove this part and add `open Classical in` above the lemmas you need `Classical` in?